tests/TestCli.cpp

```diff
@@ -168,6 +168,19 @@ void TestCli::testAdd()
     QCOMPARE(entry->password(), QString("newpassword"));
 }
 
+bool isTOTP(const QString & value) {
```

Comment:
this could be moved to `src/core/totp.cpp`, if we ever need it for other tests. You could even add some tests in `tests/TestTotp.cpp`

Reply:
Do you mean `src/totp/totp.cpp`? I'm a bit sceptic about this one, since this tests for a very specific kind of TOTP (5-6 decimal digits), while there are also other TOTPs supported by keepassxc, and the number of digits is configurable.

It looks like the UI allows you to configure:
 1. 6 digits `0123456789`;
 1. 8 digits `0123456789`;
 1. 5 digits `23456789BCDFGHJKMNPQRTVWXY`.
So I could modify the function to check for these specific parameters. Or should the function also get a `Totp::Settings` (or `Totp::Encoder`?) parameter and use the settings from there?

Reply:
maybe it's overkill to move that out of this test file... let's just leave it there for now.